src/components/ebay-pagination/index.marko

```diff
@@ -53,7 +53,17 @@ $ var range = component._getVisibleRange(items);
         key="items"
         class="pagination__items">
         <for|item, i| of=items>
-            <li hidden=(i < range.start || i > range.end)>
+            <if(range.dotsIndex === i)>
+                <li hidden=(range.hideDots)>
+                    <span
```

Comment:
An aria-label doesn't do anything without a role (implicit or explicit). A role of `img` is needed in this case. However, I think we should just remove the `a11yDotsText` attribute for now, as the text/ellipses by itself should be adequate. I'm wary of apps populating a label with something that does more harm than good!

Reply:
Done